indico/modules/designer/templates/template.html

```diff
@@ -164,13 +164,31 @@
                 </div>
             {% endif %}
         </div>
+        <div class="designer-tools-row js-hide-on-flip">
+            <div class="tool">
+                <span class="f-self-stretch">{% trans %}Layer visibility{% endtrans %}</span>
+            </div>
+            <div class="tool">
+                <label for="is-layer1-visible">1</label>
+                <button class="i-button icon-eye only-icon borderless js-is-layer-visible" id="is-layer1-visible" data-value="1"></button>
+            </div>
+            <div class="toolbar">
+                <button class="i-button icon-remove icon-only js-remove-layer" title="{% trans %}Remove last layer{% endtrans %}" disabled></button>
+                <button class="i-button icon-plus icon-only js-add-layer" title="{% trans %}Add new layer{% endtrans %}"></button>
+            </div>
+        </div>
         <div class="designer-tools-row second-row disappear">
             <div class="element-tools hidden">
                 <div class="tool">
                     <span class="selection-text f-self-stretch"></span>
                     <button class="js-remove-element i-button icon-remove icon-only text-color subtle"
                             title="{% trans %}Remove element{% endtrans %}"></button>
                 </div>
+                <div class="tool">
+                    <select id='item-layer-selector' class="js-layer-tool" data-attr="layer">
+                        <option value="1">{{ gettext('Layer {0}').format(1) }}</option>
```

Comment:
`{% trans %}Layer 1{% endtrans %}`

No need to try to use the same gettext pattern as in the JS since JS and python/jinja are separate i18n dicts anyway so they have to be translated twice in any case

Reply:
Ah, I see. But I tried hard :)

Reply:
FWIW a cleaner way to do this (if it wasn't for the separate dicts) would be this: `{% trans n=1 %}Laber {{ n }}{% endtrans %}`

(generally using `gettext` in templates directly is somewhat uncommon)

Reply:
Ok, thanks. My motivation was indeed to use the same form as in js.